client/consensus/aura/src/lib.rs

```diff
@@ -1045,7 +1052,7 @@ mod tests {
 			).expect("Starts aura"));
 		}
 
-		futures::executor::block_on(future::select(
+		task::block_on(future::select(
```

Comment:
Why change from `futures::executor::block_on` to `async_std::task::block_on`?

Reply:
Same as above.